lib/okta/directory_service.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
```

Comment:
Code coverage is a bit low for this service class

Reply:
code coverage is at 100% with latest commit #986803c